client/package.json

```diff
@@ -60,5 +60,9 @@
     "ts-node": "^8.0.3",
     "tslint": "^5.14.0",
     "typescript": "^3.2.4"
-  }
+  },
+  "description": "This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 6.1.3.",
+  "main": "index.js",
```

Comment:
why did you add an index.js here?

Reply:
I have deleted unnecessary stuff from package-lock.json and client/packag.json

I used npm install to install new version of Jquery, so it includes all the dependency and made changes into the files. Later I understood this is not required as at the time of first run(npm install) all the dependency definition are already there, so need to change that.